common/ayon_common/distribution/data_structures.py

```diff
@@ -240,7 +252,7 @@ def from_dict(cls, installer_info):
             sources=sources,
             unknown_sources=unknown_sources,
             checksum=installer_info["checksum"],
-            checksum_type=installer_info["checksumAlgorithm"],
+            checksum_algorithm=installer_info.get("checksumAlgorithm"),
```

Comment:
Should we default to `, "sha256"` here too?

Reply:
```suggestion
            checksum_algorithm=installer_info.get("checksumAlgorithm", "md5"),
```